src/main/scala/se/apogo/kdom/Domino.scala

```diff
@@ -2,3 +2,58 @@ package se.apogo.kdom
 
 case class Domino(number: Int, tile1: Tile, tile2: Tile) {
 }
+
+object Domino {
+  private val Dominoes: Seq[Domino] = Seq(
```

Comment:
More idiomatic Scala would be to  skip `private`, rename to `allDominoes` and remove `def allDominoes: Seq[Domino] = Dominoes`.

Reply:
Fixed and pushed with amend.

Yep, I see why that is more idiomatic, less boilerplate.